src/cli/init.ts

```diff
@@ -48,6 +64,24 @@ export default class InitCommand extends BaseCommand {
       this.spinner.start('Set Offer ID')
       this.offerId = offerId
       this.spinner.stop()
+
+      // Peer identity
+      this.spinner.start('Generating Peer Identity')
+      const store = getObject()
+      const peerId = (await PeerId.create({
+        keyType: this.parsedArgs.flags.keyType,
+        bits: this.parsedArgs.flags.keySize
+      })).toJSON()
+
+      store.peerId = peerId.id
+      store.peerPubKey = peerId.pubKey
+      store.peerPrivKey = peerId.privKey
+      this.spinner.stop()
+
+      const uiUrl = config.get<string>('uiUrl')
+      this.log(`Create Offer here: ${uiUrl.replace(PEER_ID_PLACEHOLDER, peerId.id)}`)
```

Comment:
```suggestion
      this.log(`Create Offer here: ${uiUrl.replace(PEER_ID_PLACEHOLDER, peerId.toB58String()}`)
      this.log(`Or input your PeerId into the form: ${peerId.toB58String()}`)
```
I think we should give the option, some people don't like copying or clicking links.

Reply:
What option?